src/useRender.js

```diff
@@ -0,0 +1,6 @@
+import { useReducer, useCallback } from 'react'
+
+export default () => {
+  const [, forceRender] = useReducer(s => s + 1, 0)
```

Comment:
What is this doing here?

Reply:
it is a trick I learned from the `react-redux` code base to force a component to refresh. The issue here is that the `targets` object is not part of the component lifecycle or state, so when it changes it will not update the UI.  This is allows the component to be re-rendered on command.

Reply:
cool beans